src/tests/java/htsjdk/samtools/sra/SRAIndexTest.java

```diff
@@ -64,15 +62,13 @@ public void testLevelSize() {
 
     @Test
     public void testLevelForBin() {
-        if (!SRAAccession.isSupported()) return;
-
         SRAIndex index = getIndex(DEFAULT_ACCESSION);
```

Comment:
I think I got them all.  For some reason my intellij isn't reporting them in the test files and I can't figure out why not...